server/middlewares/userValidator.js

```diff
@@ -0,0 +1,125 @@
+        .blacklist(' ')
+        .customSanitizer(value => makeLowerCase(value))
+        .custom(async (value) => {
+          const foundUser = await user.find({ where: { email: value } });
+          if (foundUser) {
+            return Promise.reject(new Error('Email already exists'));
+          }
+        }),
+      check('firstName')
+        .exists().withMessage('Firstname is missing')
+        .not()
+        .isEmpty({ ignore_whitespace: true })
+        .withMessage('Firstname cannot be blank')
+        .isAlpha()
+        .withMessage('Firstname can only contain alphabets')
+        .isLength({ min: 2, max: 20 })
+        .withMessage('Firstname must be at least 2 characters, and maximum 20')
+        .blacklist(' ')
+        .customSanitizer(value => capitaliseFirstLetter(value)),
+      check('lastName')
+        .exists().withMessage('Lastname is missing')
+        .not()
+        .isEmpty({ ignore_whitespace: true })
+        .withMessage('Lastname cannot be blank')
+        .isAlpha()
+        .withMessage('Lastname can only contain alphabets')
+        .isLength({ min: 2, max: 20 })
+        .withMessage('Lastname must be at least 2 characters, and maximum 20')
+        .blacklist(' ')
+        .customSanitizer(value => capitaliseFirstLetter(value)),
+      check('password')
+        .exists().withMessage('Password is missing')
+        .not()
+        .isEmpty({ ignore_whitespace: true })
+        .withMessage('Password cannot be blank')
+        .isLength({ min: 6, max: 20 })
+        .withMessage('Password must be at least 6 characters')
+        .blacklist(' '),
+      check('confirmPassword')
+        .exists().withMessage('Password confirmation is missing')
+        .not()
+        .isEmpty({ ignore_whitespace: true })
+        .withMessage('Password confirmation cannot be blank')
+        .isLength({ min: 6, max: 20 })
+        .withMessage('Password must be at least 6 characters')
+        .blacklist(' ')
+        .custom((value, { req }) => {
+          if (value !== req.password) throw new Error('Both passwords must match');
+          return true;
+        }),
+      check('avatarUrl')
+        .exists().withMessage('Url for avatar doesn\'t exist')
+        .not()
+        .isEmpty({ ignore_whitespace: true })
+        .withMessage('AvatarUrl cannot be blank')
+        .isURL()
+        .withMessage('AvatarUrl must be a valid URL string')
+        .blacklist(' '),
+      check('verified')
+        .exists().withMessage('Verification status is missing')
+        .not()
+        .isEmpty({ ignore_whitespace: true })
+        .withMessage('Verification status cannot be blank')
+        .isBoolean()
+        .withMessage('Verification status must be of boolean type')
+        .blacklist(' '),
+    ];
+  }
+
+  /**
+  * Adds validations to the user routes
+  * @returns {array} an array containing the Check API validations
+  * @memberof Validation
+  */
+  static validateLogin() {
+    return [
+      check('email')
```

Comment:
email validation schema for sign up and email validation schema for login is the same. You have the same similar schema definitions on the same page, why not assign them to a variable, and call the variables when required, so that you don't repeat yourself.